lighthouse-core/scripts/i18n/collect-strings.js

```diff
@@ -22,9 +30,25 @@ const ignoredPathComponents = [
   '-renderer.js',
 ];
 
+const defaultDescriptions = {
+  failureTitle: 'Shown to users as the title of the audit when it is in a failing state.',
+};
+
+// @ts-ignore - waiting for esprima types, see above TODO
+function computeDescription(ast, property, startRange) {
+  const endRange = property.range[0];
```

Comment:
do we want to prepend `[ICU Syntax]​` here if it is?

Reply:
@paulirish was against that since it only applies to ones that have the template and there are only ~5